vhdl-ext-nav.el

```diff
@@ -192,7 +193,12 @@ Kill the buffer if there is only one match."
       (cond ((eq num-matches 1)
              (xref-push-marker-stack vhdl-ext-jump-to-parent-module-point-marker)
              (next-error)
-             (kill-buffer (current-buffer))
```

Comment:
The `progn` here would not be needed since all but the first statement belong to the else clause:
```elisp
             (xref-push-marker-stack vhdl-ext-jump-to-parent-module-point-marker)
             (next-error)
             (if (> vhdl-ext-jump-to-parent-entity-starting-windows 1)
                 (kill-buffer (current-buffer))
               (other-window 1)
               (delete-window))
             (message "Jump to only match for [%s] @ %s" entity-name dir))
            ((> num-matches 1)
             (xref-push-marker-stack vhdl-ext-jump-to-parent-module-point-marker)
```